AdvPromptSet/main.py

```diff
@@ -0,0 +1,38 @@
+import numpy as np
+import os
+import pandas as pd
+from utils import process_jigsaw_1
+from utils import process_jigsaw_2
+from utils import process_jigsaw_all
+from utils import check_shape
+
+
+if __name__ == "__main__":
```

Comment:
Maybe give a docstring or just a comment at the top saying what this script does? People might see this script without seeing the README first

Reply:
Make sense! added